### PR TITLE
Add initial adopters list

### PR DESCRIPTION
**Pull request by kubukoz**
Very much WIP - I only added a few companies that (I guess) use Cats in prod. Please suggest others!

### CODE REVIEW

**Comment by kailuowang** [comment]
I love the idea. Would you add [iHeartRadio](iheartradio.com) to the list please?

File: README.md [hunk 1]
```diff
@@ -183,6 +183,14 @@ Any binary breaking changes will require a *MAJOR* version bump, which we will b
 cautious about. We will also consider using `organization` and package name for major 
 versioning in the future. But that decision is yet to be made. 
 
+### Adopters
+
+Here's a (non-exhaustive) list of companies that use Cats in production. Don't see yours? [Add it in a PR!](https://github.com/typelevel/cats/edit/master/README.md)
+
+- Codecentric
```
[the remark this answers is not in the document]
Comment (kubukoz) [reply README.md]: I was also considering having a table with `name | link | usage` that'd highlight how much the company uses cats - is it only `Validated` and monad transformers? Is it `cats-effect` and tagless final?

But I guess links will do for now

**Comment by hamishdickson** [comment]
Love this - we use cats almost everywhere at [DriveTribe](http://drivetribe.com/)

File: README.md [hunk 2]
```diff
@@ -183,6 +183,16 @@ Any binary breaking changes will require a *MAJOR* version bump, which we will b
...
+- e.near
+- iHeartRadio
+- Stripe
+- Verizon
```
Comment (rossabaker) [inline README.md]: Not at the time I left.  It may be true now, or in another department.

**Comment by kubukoz** [comment]
(I'll add links when we have a larger list)

File: README.md (hunk 2 above)
Comment (kubukoz) [reply README.md]: I'll remove for now, then.

**Comment by rossabaker** [comment]
This PR could get unwieldy if everybody comments instead of sending a PR.  Feel free to cut it off at whatever arbitrary point you see fit and invite others to add themselves.

File: README.md (hunk 2 above)
Comment (rossabaker) [reply README.md]: We employed several committers and our open source libraries and Nelson are all converted, so we left a solid Cats legacy.  We just never got there ourselves. :cry:

**Comment by jakubjanecek** [comment]
Could you please add Avast (avast.com) to the list? Thank you.

**Comment by ChristopherDavenport** [comment]
This is great! 

We are also using it in production here at [Banno Group inside of Jack Henry & Associates](https://banno.com/)

**Comment by MansurAshraf** [comment]
Please add FEAR team at Apple Inc. Here is the job posting https://news.ycombinator.com/item?id=16969118

**Comment by gatorcse** [comment]
Not for this PR, but might be cool to get a nice logo banner on the main site highlighting some of the bigger users. See [hapijs's](https://hapijs.com) site if you don't understand what I'm talking about.

**Comment by gabro** [comment]
Also, buildo (www.buildo.io). Thanks!

**Comment by igstan** [comment]
Nice party 😸! Please also add [eloquentix.com](http://eloquentix.com/) and [eon.com](https://www.eon.com/). Thanks!

**Comment by gatorcse** [comment]
[47 Degrees](https://www.47deg.com)

**Comment by eperinan** [comment]
Awesome iniciative!

We are using it in several projects in www.spotahome.com.

Thanks for all!

**Comment by kubukoz** [comment]
We could extend this idea and add a page on the website with some quotes from key technical people (CTOs, team leads etc.) about how Cats and TL libs help them in daily work :)

**Comment by javierarrieta** [comment]
Will create a PR and add Zalando when this is merged, thanks!

**Comment by kubukoz** [comment]
@javierarrieta I've already been asked by @etorreborre to add it ;)

**Comment by notxcain** [comment]
[Evotor Marketplace](https://market.evotor.ru)

**Comment by kailuowang** [comment]
I am going to merge as soon as builds go green.

**Comment by kubukoz** [comment]
🎉🎉🎉

**Comment by omervk** [comment]
I'm happy to share that we at [WeWork](https://www.wework.com/) are part of this list.

**Comment by LukaJCB** [comment]
@omervk go right ahead :)

**Pull request merged**